src/App.jsx

```diff
@@ -1,33 +1,42 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
+export const App = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer
+    .filter(movie => movie.title.toLowerCase().includes(query)
+  || movie.description.toLowerCase().includes(query));
```

Comment:
It will be more cleaner to move this logic to separate function outside of component
```suggestion
  const [query, setQuery] = useState('');
  
  const visibleMovies = moviesFromServer
    .filter(movie => movie.title.toLowerCase().includes(query)
      || movie.description.toLowerCase().includes(query));
```